src/merge/mod.rs

```diff
@@ -95,7 +98,7 @@ pub fn merge_lines<'a>(a: &'a str, o: &'a str, b: &'a str) -> MergeResult<&'a st
                     Resolved(y) => y.into_iter(),
                     _ => unreachable!()
                 })
-                .flat_map(|x| vec![x, "\n"].into_iter())
+                .intersperse("\n")
```

Comment:
While this is an improvement, I would prefer not to add an additional dependency for this single line. Can you revert this, please? :)

Reply:
Unfortunately, https://doc.rust-lang.org/std/slice/trait.SliceConcatExt.html is not yet stable, and I see no nice way to put \n's _between_ the items. Maybe you know one?

Also, this is highly readable.

Reply:
In between! Of course :) Then I'd suggest using join, again to avoid introducing an extra dependency:

                .flat_map(...)
                .collect::<Vec<_>>()
                .join("\n")

Reply:
done (although you still pull in itertools anyway via bart_derive)